prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+ Partition     Min size    Best size    Mandatory
+============  ==========  ===========  =======================================
+ ``/boot``      256Mib      500Mib      Optional, partition used to boot
+ ``/``           4Gi        120Gib      Yes, the partition were system install
+ ``swap``       100Mib      16Gib       Yes, the partition for virtual ram
+============  ==========  ===========  =======================================
+
+The Data drive storage (DDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The data storage disk will be partially managed by the |omv| after installed,
+and only in customized installation can be a partiton of the same system drive
+storage.
+
+Each data storage drive will be managed, but regardless of the partitioning
+scheme, all data drive partitions will be mounted at the ``/srv/`` path.
+The **data storage drive must be a different disk drive than the system disk drive**
+where the operating system is installed. **One partition of each data
+drive can handle one or many shared resources.**
+
+==============  ==========  ===========  ========================================
+ Partition       Min size    Best size    Mandatory
+==============  ==========  ===========  ========================================
+ ``/srv/<*>``     100Mib       bigger     Yes, size as need! example 128Gib
+==============  ==========  ===========  ========================================
+
+Technical notes
+---------------
+
+Take into **consideration that as more exquisite the customization as less supported
+could be.**.
+
+|omv| allows for minimal deployment and use of the system with very minimal
+hardware/software requirements, at the cost of the key knowledge, but other NAS
+systems allow ease of use at the cost of high requirements. |omv| allows both as
+well in a well balanced use cases! [7]_
+
+About minimal or custom setups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It can be noted that |omv| system can be installed on just 4Gib of system partition,
+with no more than 1Gib of RAM and using a WiFi or USB connection to access it
+remotely (cos is a NAS system of course), on any small device such as Raspberry
+ones which are ARM, or old i386s machines no matter is those are 32 or 64 bits, 
+but of course as more exquisite the configuration more knowledge will be need
+as per `issue comment #131 <https://github.com/openmediavault/openmediavault-docs/issues/131#issuecomment-2546765841>`_.
+
+Technically OMV can be installed on a single storage disk, this is possible if
+it is done on a previously configured Debian system, with a free partition apart
+from the 3 necessary system partitions, previously formatted and configured.
+Of course this is only possible for skilled linux users.
+
+Drive Storage technical details
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+System Drives are not managed as same of Data Drives. System drives are not so
+intensively used, but Data Drives will need triks to extend the useful life.
```

Comment:
triks ??? Is this a typo?

Reply:
triks was a typo, fixed with commit c441ec7d9649da3fb60e9f23635c041b62f2a54c but treated was not!